### PR TITLE
DOCS-147 - Add hidden instructions for Cloud SOAR (Kanso)

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request adds instructions for the new UI (Kanso) to Cloud SOAR articles in hidden text. For example:
```
<!--Kanso 
hiddent text
 Kanso-->
```

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[DOCS-147](https://sumologic.atlassian.net/browse/DOCS-147)

### CODE REVIEW

File: docs/cloud-soar/automation.md [hunk 1]
```diff
@@ -10,22 +10,24 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 The **Automation** section contains the configuration tools for Cloud SOAR's automation and orchestration features. These tools include Cloud SOAR's Open Integration Framework (OIF), automation rules sets, and playbook editor as well as incoming event details.
 
-To access this section, click the cog icon (<img src={useBaseUrl('img/cloud-soar/cog.png')} alt="cog menu" width="20"/>) > **Automation**.
```
Comment (kimsauce) [inline docs/cloud-soar/automation.md:13]: It looks like you've eliminated these references to the cog.png image in these docs. If so, I'd also delete the image itself from static/img/cloud-soar/cog.png.

File: docs/platform-services/automation-service/automation-service-audit-logging.md [hunk 2]
```diff
@@ -15,14 +15,14 @@ Audit logging for the Automation Service uses the same logging as Cloud SOAR, si
 
 ## Search the Audit Event Index
 
-Searching the Audit Event Index is the same as running a normal search against your ingested data. You specify the `_index` metadata field with one of these values: 
+Searching the Audit Event Index is the same as running a normal search against your ingested data. You specify the `_index` metadata field with one of these values:
```
Comment (kimsauce) [inline docs/platform-services/automation-service/automation-service-audit-logging.md:18]:
```suggestion
Searching the Audit Event Index is the same as running a normal search against your ingested data. You specify the `_index` metadata field with one of these values: 
```

File: docs/platform-services/automation-service/automation-service-audit-logging.md [hunk 3]
```diff
@@ -126,8 +126,6 @@ Here is an example `PlaybookExecutionStarted` event log.
 }
 ```
 
-## Index retention period 
+## Index retention period
```
Comment (kimsauce) [inline docs/platform-services/automation-service/automation-service-audit-logging.md:129]:
```suggestion
## Index retention period 
```

File: docs/platform-services/automation-service/automation-service-audit-logging.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ Searching the Audit Event Index is the same as running a normal search against y
 
 ## Automation Service audited events
 
-To search for audit events for a specific feature, use the metadata field `_sourceCategory` with its corresponding value. 
+To search for audit events for a specific feature, use the metadata field `_sourceCategory` with its corresponding value.
```
Comment (kimsauce) [inline docs/platform-services/automation-service/automation-service-audit-logging.md:25]:
```suggestion
To search for audit events for a specific feature, use the metadata field `_sourceCategory` with its corresponding value. 
```

**Review by kimsauce — APPROVED** [review]
Made [one commit](https://github.com/SumoLogic/sumologic-documentation/pull/4041/commits/46c084213b864a0ee5c864267194e3c4da96f6b0) and have one additional outstanding comment

File: docs/cloud-soar/automation.md (hunk 1 above)
Comment (jpipkin1) [reply docs/cloud-soar/automation.md:13]: Good call, @kimsauce. I made sure the image wasn't being used anywhere, and then I removed it (see commit "[Remove /img/cloud-soar/cog.png](https://github.com/SumoLogic/sumologic-documentation/pull/4041/commits/0b6d3de3cb467026baae448ffb1c360d03dd349d)").

**Pull request merged**